<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="table_A">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>321</width>
      <height>251</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QTableWidget" name="table_B">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>40</y>
      <width>121</width>
      <height>251</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QTableWidget" name="table_x">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>40</y>
      <width>121</width>
      <height>251</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Matrix A:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Matrix B:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>10</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Vector x:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push_button_solve">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>300</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Solve</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <widget class="QMenu" name="menu_edit_A_matrix">
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Edit A matrix</string>
    </property>
    <addaction name="action_load_A"/>
    <addaction name="action_save_A"/>
    <addaction name="separator"/>
    <addaction name="action_add_new_row_to_A"/>
    <addaction name="action_add_new_column_to_A"/>
    <addaction name="action_remove_row_from_A"/>
    <addaction name="action_remove_column_from_A"/>
    <addaction name="separator"/>
    <addaction name="action_randomize_A"/>
    <addaction name="action_create_matrix_A"/>
   </widget>
   <widget class="QMenu" name="menu_edit_B_matrix">
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Edit B matrix</string>
    </property>
    <addaction name="action_load_B"/>
    <addaction name="action_save_B"/>
    <addaction name="separator"/>
    <addaction name="action_add_new_row_to_B"/>
    <addaction name="action_remove_row_from_B"/>
    <addaction name="separator"/>
    <addaction name="action_randomize_B"/>
    <addaction name="action_create_matrix_B"/>
   </widget>
   <widget class="QMenu" name="menu_soultion_x">
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Soultion x</string>
    </property>
    <addaction name="action_save_x"/>
   </widget>
   <addaction name="menu_edit_A_matrix"/>
   <addaction name="menu_edit_B_matrix"/>
   <addaction name="menu_soultion_x"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_load_A">
   <property name="text">
    <string>Load matrix</string>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
  </action>
  <action name="action_save_A">
   <property name="text">
    <string>Save matrix</string>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
  </action>
  <action name="action_add_new_row_to_A">
   <property name="text">
    <string>Add new row</string>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
  </action>
  <action name="action_add_new_column_to_A">
   <property name="text">
    <string>Add new column</string>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
  </action>
  <action name="action_remove_row_from_A">
   <property name="text">
    <string>Remove row</string>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
  </action>
  <action name="action_remove_column_from_A">
   <property name="text">
    <string>Remove column</string>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
  </action>
  <action name="action_load_B">
   <property name="text">
    <string>Load from file</string>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
  </action>
  <action name="action_save_B">
   <property name="text">
    <string>Save to file</string>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
  </action>
  <action name="action_add_new_row_to_B">
   <property name="text">
    <string>Add row</string>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
  </action>
  <action name="action_remove_row_from_B">
   <property name="text">
    <string>Remove row</string>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
  </action>
  <action name="action_randomize_A">
   <property name="text">
    <string>Randomize...</string>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
  </action>
  <action name="action_randomize_B">
   <property name="text">
    <string>Randomize...</string>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
  </action>
  <action name="action_save_x">
   <property name="text">
    <string>Save to file</string>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
  </action>
  <action name="action_create_matrix_A">
   <property name="text">
    <string>Create matrix...</string>
   </property>
  </action>
  <action name="action_create_matrix_B">
   <property name="text">
    <string>Create Matrix...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
